pkg/manager/manager.go

```diff
@@ -144,6 +145,7 @@ func (m *SupportBundleManager) Run() error {
 }
 
 func (m *SupportBundleManager) phaseInit() error {
+	m.ExtraBundleCollectors = []string{"Harvester"}
```

Comment:
Let's add a command line flag for this and default to nothing.
Longhorn will also use the tool and we need to be generic in default cases.

Reply:
yap, I would add another commit for fix this

Reply:
fixed, please check it again.